zend_abstract_interface/symbols/lookup.c

```diff
@@ -142,16 +120,16 @@ static inline zend_class_entry *zai_symbol_lookup_class_impl(zai_symbol_scope_t
         } break;
 
         case ZAI_SYMBOL_SCOPE_NAMESPACE: {
-            zai_str key = zai_symbol_lookup_key(scope, name, true);
+            zai_string key = zai_symbol_lookup_key(scope, name, true);
 
-            if (key.ptr) {
```

Comment:
Based on the previous implementation of `zai_symbol_lookup_key`, this `key.ptr` should always evaluate to true. If so, we are able to remove this `if` check and just always do the body, right?

Reply:
right :-)